worker/src/index.ts

```diff
@@ -206,6 +208,21 @@ export async function importPackage (
   })
 }
 
+// In Windows system exFAT drive, symlink will result in error.
+function currentDriveIsExFAT (dir: string): boolean {
+  const currentDrive = `${dir.split(':')[0]}:`
+  const output = execSync(`wmic logicaldisk where "DeviceID='${currentDrive}'" get FileSystem`).toString()
+  const lines = output.trim().split('\n')
+  const name = lines.length > 1 ? lines[1].trim() : ''
+  return name === 'exFAT'
+}
```

Comment:
This function is a duplication of the same function in plugin-commands-doctor.

Reply:
Yes, I have tried to reuse it before. However, if it is reused, maybe we need to remove the parameters. Otherwise, the GitHub code scan will show the same warning message as before.